app/components/collections/show/deposits_component.html.erb

```diff
@@ -1,6 +1,4 @@
-<div class="tab-pane fade" id="deposits-pane" role="tabpanel" aria-labelledby="deposits-tab" tabindex="0">
```

Comment:
Removed the tab part of this component so that it only renders the deposits content.